src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.launchcode.AroundTownServer.data;
+
+import com.launchcode.AroundTownServer.models.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends CrudRepository<User, Integer> {
+    User findByUsername(String username);
```

Comment:
Unsure if line 9 is necessary for this feature, may b e able to delete and still have function

Reply:
I can't find a good resource right now to explain why, but it is necessary I'm pretty sure!